mappings/net/minecraft/client/render/TexturedRenderLayers.mapping

```diff
@@ -44,3 +46,13 @@ CLASS net/minecraft/class_4722 net/minecraft/client/render/TexturedRenderLayers
 	METHOD method_24074 getEntityCutout ()Lnet/minecraft/class_1921;
 	METHOD method_24076 getEntityTranslucentCull ()Lnet/minecraft/class_1921;
 	METHOD method_29382 getItemEntityTranslucentCull ()Lnet/minecraft/class_1921;
+	METHOD method_33081 getBannerPatternTextureId (Lnet/minecraft/class_2582;)Lnet/minecraft/class_4730;
```

Comment:
Is there any difference between these private methods and the public ones?

Reply:
The private ones create a new instance of `SpriteIdentifier`, the public ones fetch them from a map (which is populated using the private method).

Reply:
I think given their function is not identical they should be named separately rather than qualified by `Private`. Could use `create` instead of `get`, potentially